src/mmw/apps/modeling/calcs.py

```diff
@@ -191,6 +192,41 @@ def get_weather_simulation_for_project(project, category):
     }, errs
 
 
+def get_available_simulations_for_aoi(area_of_interest):
+    """
+    Get dict of eligible weather categories for an area of interest.
+
+    Current categories are NASA_NLDAS_2000_2019, RCP45_2080_2099 and
+    RCP85_2080_2099, and only support shapes within DRB and PA. We check
+    if the two nearest weather stations for the given area of interest are
+    available in each of these, and if so, enable them.
+    """
+
+    wss = nearest_weather_stations([(None, None, area_of_interest)])
+    available_sims = WeatherType.simulations.copy()
+    groups = WeatherType.simulation_groups.copy()
+
+    # Remove any simulation that does not have values for selected stations
+    for sim in WeatherType.simulations:
+        if sim in available_sims:
+            for ws in wss:
+                url = settings.WEATHER_DATA_BUCKET_URL.format(
+                          category=sim, station=ws.station)
+                res = requests.head(url)
+                if not res.ok:
+                    available_sims.remove(sim)
+                    break
```

Comment:
Haven't written a `break` statement in a while. We're trying to disqualify simulations as soon as possible, so break out of the loop once that is established.